tenable/io/v3/mssp/logos/api.py

```diff
@@ -0,0 +1,229 @@
+                the field.
+            fields (list):
+                list of fields to retrieve.
+        Returns:
+            :obj:`LogosIterator`:
+                An iterator that handles the page management of the requested
+                records.
+        Examples:
+            Todo
+        '''
+        raise NotImplementedError('Search method will be implemented later')
+
+    def details(self, logo_id: UUID) -> Dict:
+        '''
+        Returns Logo details
+
+        Args:
+            logo_id (UUID):
+                The UUID of the logo
+        Returns:
+            :obj:`dict`:
+                Return the logo details.
+
+        Example:
+            >>>> tio.v3.mssp.logos.details(
+            ....    '61a36add-d29b-4a52-bbce-c8215952ede5')
+        '''
+        return self._get(f'{logo_id}')
+
+    def update(self, logo_id: UUID, **kw) -> str:
+        '''
+        Returns Logo details
+
+        Args:
+            logo_id (UUID):
+                The UUID of the logo
+        Returns:
+            :obj:`str`:
+                Return the logo_id.
+
+        Example:
+            >>>> fobj = open('/opt/test_image.png', 'rb')
+            ....     update_resp = tio.v3.mssp.logos.update(
+            ....     '0cba902a-bd11-4481-bd28-999c88ffe22f',
+            ....     name='update_name.png',
+            ....     logo=fobj)
+        '''
+        logo_dict = {}
+        if kw.get('name'):
+            logo_dict['name'] = kw['name']
+        if kw.get('logo'):
+            logo_dict['logo'] = kw['logo']
+        payload = {'files': logo_dict}
+        return self._patch(f'{logo_id}', **payload)['id']
+
+    def delete(self, logo_id: UUID) -> None:
+        '''
+        Deletes Logo for the gived logo_id
+
+        Args:
+            logo_id (UUID):
+                The UUID of the logo
+        Returns:
+            None
+
+        Example:
+            >>>> tio.v3.mssp.logos.delete(
+            ....    '61a36add-d29b-4a52-bbce-c8215952ede5')
+        '''
+        self._delete(f'{logo_id}')
+
+    def assign_logos(self, logo_id: UUID, account_ids: List) -> None:
+        '''
+        Assigns a logo to one or more customer accounts in the Tenable.io
+        MSSP Portal.
+
+        Args:
+            logo_id (UUID):
+                The UUID of the logo
+            account_ids (List):
+                list of accoint_ids to assign logos to.
+
+        Returns:
+            None
+
+        Example:
+            >>>> tio.v3.mssp.logos.assign_logos(
+            ....    'a39f6b74-9b7f-4372-a7ac-a2a4bcb8dbad',
+            ....        ['0fc4ef49-2649-4c76-bfa7-c181be3adf26'])
+        '''
+        payload = {'account_ids': account_ids, 'logo_id': logo_id}
+        self._schema.load(self._schema.dump({
+            'account_ids': account_ids,
+            'logo_id': logo_id
+        }
+        ))
+        self._api.put('api/v3/mssp/accounts/logos', json=payload)
+
+    def download_png(
```

Comment:
Please update the method declaration like this:
https://github.com/dhvanilshah-tenable/pyTenable/pull/12/files#diff-f08a9da0fb05c1c53ae0da456a95945971dfbf2362c0fa30e541363d3728e227R167